### DISCUSSION TITLE
Chronos Bolt Fine Tuning Base Version

### DISCUSSION THREAD

**Discussion by gknair** [amazon-science/chronos-forecasting#260] [Q&A]
I have been evaluating the performance of the Chronos Bolt base models for data over the past two years of hourly data, and have observed satisfactory results for only one month out of the 24. To improve this, I have been experimenting with fine-tuning the model throughout 2023. This involves using hourly data comprising approximately 8,460 data points, with a context window of 1,024 and a prediction window of 64.

During discussions, it was recommended to fine-tune using AutoGluon. Could you provide more insights into the test and validation window configurations used by AutoGluon during fine-tuning? I understand that the learning rate and training steps can be adjusted, but I am curious to know if there are options to modify the test and validation window sizes as well

**Comment by shchur** [comment]
Hi @gknair, you will most likely need to experiment with several different configurations (learning rate, number of epochs, etc) to find the one that works best for your task. If your dataset contains just one time series with a long history, I recommend increasing the number of validation windows with `num_val_windows` and setting `refit_every_n_windows=None` to ensure that the model is only trained once and evaluated on all of the validation windows.

```python
predictor = TimeSeriesPredictor(...)
predictor.fit(train_data, hyperparameters={"Chronos": {...}}, num_val_windows=10, refit_every_n_windows=None)
```
